src/views/List.jsx

```diff
@@ -36,12 +109,60 @@ export function List({ data, listPath }) {
 		return () => clearTimeout(getItem);
 	}, [searchTerm, data]);
 
+	const addItemForm = () => {
+		return (
+			<>
+				{alertText(submitted)}
+				<form onSubmit={handleSubmit}>
+					<div className="newItemInputs">
+						<label htmlFor="itemName">Item name: </label>
```

Comment:
As of now, with many content and input box on list page. Could change label from "Item name" to "add Item" or add a section header , THIS IS AN ADD ITEM SECTION" to tell people to add item.

Reply:
Wrapped the form in a section tag along with a header.
The label needs to tell the user what the input accepts so I think "Item name" / "Enter item name" is more appropriate.